schedule/yast/transactional_server/create_hdd_transactional_server.yaml

```diff
@@ -67,8 +67,11 @@ schedule:
     - "{{reconnect_mgmt_console}}"
     - "{{grub_test}}"
     - installation/first_boot
-    - console/hostname
```

Comment:
Any reason to move `console/hostname`?

Reply:
I moved it [after the modules with zypper](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/e390cd321477d419eeb1fdcfccb3a33dc8bc9d9b/schedule/yast/transactional_server/create_hdd_transactional_server.yaml#L74) just to be consistent with the similar test suite [create_hdd_gnome](https://openqa.opensuse.org/tests/1602616). I guess it will not harm.